mashup-example/Qlik-Sense-Desktop/Using-Build-Tool/.gitignore

```diff
@@ -0,0 +1,24 @@
+*.log
```

Comment:
All of these shouldn't be needed

Reply:
okay, removing this .gitignore file?

Reply:
No I mean I don't think we need to ignore all of this

Reply:
okay